etc/docker/php/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+chown -R www-data:www-data -R app/cache app/logs
```

Comment:
It might be! :) That makes sense, there's no image being created after the entrypoint.